python/pyspark/sql/connect/conf.py

```diff
@@ -49,6 +49,20 @@ def set(self, key: str, value: Union[str, int, bool]) -> None:
 
     set.__doc__ = PySparkRuntimeConfig.set.__doc__
 
+    def _set_all(self, configs: Dict[str, Union[str, int, bool]], silent: bool) -> None:
+        conf_list = []
+        for key, value in configs.items():
+            if isinstance(value, bool):
+                value = "true" if value else "false"
+            elif isinstance(value, int):
+                value = str(value)
+            conf_list.append(proto.KeyValue(key=key, value=value))
+        op_set = proto.ConfigRequest.Set(pairs=conf_list, silent=silent)
+        operation = proto.ConfigRequest.Operation(set=op_set)
+        result = self._client.config(operation)
```

Comment:
self._client.config might raise errors if I remember correctly?

Reply:
yes, when `silent` is not set or set False, and trying to set a static config